src/adaptive_graph_of_thoughts/config.py

```
@@ -321,7 +323,68 @@
+    mcp_transport_type: str = "http"
+    mcp_stdio_enabled: bool = True
+    mcp_http_enabled: bool = True
+    log_level: str = "INFO"
+
+
+class RuntimeSettings(BaseSettings):
+    """Combined settings object used by the application."""
+
+    app: AppSettingsModel = AppSettingsModel()
+    neo4j: Neo4jSettingsModel = Neo4jSettingsModel()
+    asr_got: dict[str, Any] = field(default_factory=dict)
```

Pylint (3.3.7)

[error] Invalid usage of field(), it should be used within a dataclass or the make_dataclass() function.

(E3701)